ci_script.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+mkdir -p /home/catkin_ws/src
+cp -r rvip /home/catkin_ws/src
+cd /home
+git clone https://github.com/STORM-IRIT/OpenGR.git
+cd OpenGR
+git reset --hard 0967cd880950b35786b8fd098837c9eb1fe2aca4
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release ..
+make install
```

Comment:
@ipa-mdl: would you have 5 mins to suggest a possible adaptation of the `industrial_ci` config in this repo to mimic these lines?

Reply:
I believe we can handle this by adding a `.travis.rosinstall` file to the repository.

Sorry for bothering you @ipa-mdl.